lib/spdy-transport/stream.js

```diff
@@ -271,6 +271,9 @@ Stream.prototype._handleWindowUpdate = function _handleWindowUpdate(frame) {
   var state = this._spdyState;
 
   state.window.send.update(frame.delta);
+  if (state.window.send.current > 2147483647) {
```

Comment:
Considering that this is duplicated, I wonder if it could be moved to `window.js` file? Perhaps some new event may be used to indicate this kind of errors.

I would also pass this constant to the Window on initialization as a part of the options, and would make it inherit it on clone.